src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -159,7 +159,7 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
                             "${qnIdx}", Integer.toString(qnIdx),
                             "${responseIdx}", Integer.toString(responseIdx),
                             "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled=\"disabled\"",
+                            "${disabled}", sessionIsOpen ? "" : "disabled=\"\"",
```

Comment:
Regarding `ElementTag`, you should only have to do a `setAttribute(attributeName, null)` for the former version? I am pretty sure I accounted for that case after some reading up several months ago... Any `=""` around created by `ElementTag` is likely due to legacy issues during the migration where we tried to minimize the amount of HTML changes.

Would it be worth modifying `HtmlHelper` to support it?